snforge_std/src/trace.cairo

```diff
@@ -2,48 +2,55 @@ use core::result::ResultTrait;
 use core::starknet::testing::cheatcode;
 use core::starknet::ContractAddress;
 
+/// Tree-like structure which contains all of the calls and sub-calls along with the results
 #[derive(Drop, Serde, PartialEq)]
 struct CallTrace {
-    entry_point: CallEntryPoint,
-    nested_calls: Array<CallTrace>,
+    entry_point: CallEntryPoint, /// The topmost entry point of this trace
+    nested_calls: Array<CallTrace>, /// All the calls that happened in the context of `entry_point`
     result: CallResult
 }
 
+/// A single function entry point summary
```

Comment:
```suggestion
/// A summary of call to a contract entry point
```

Reply:
What if it's a library call, it's not a strictly call to a contract per-se

Reply:
call  \in {contract call, library call} imo, but if you think ur version is better feel free to resolve